src/controllers/bulk/bulkcontroller.h

```diff
@@ -76,6 +76,9 @@ class BulkController : public Controller {
     unsigned short product_id;
     unsigned char in_epaddr;
     unsigned char out_epaddr;
+#ifdef _WIN32
+    unsigned int interface_number;
+#endif
     QString manufacturer;
     QString product;
```

Comment:
```suggestion
    unsigned short mProductId;
    unsigned char mInEndpointAddr;
    unsigned char mOutEndpointAddr;
#ifdef _WIN32
    unsigned int mInterfaceNumber;
#endif
    QString mManufacturer;
    QString mProduct;
```

Reply:
I've renamed the fields with `m_<lower>` as that's what I used in other places